### PR TITLE
Stylitics Partnership Docs

### PR DESCRIPTION
**Pull request by KellieHawks**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @josh-mccrowell-braze and @KellieHawks as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @josh-mccrowell-braze or @KellieHawks for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@josh-mccrowell-braze</td>
    <td>Monolith Deployments<br>Quality Infrastructure<br>Platform Infrastructure<br>Datalake<br>SDKs</td>
  </tr>
  <tr>
    <td>@kelliehawks</td>
    <td>Currents<br>Internal Tools<br>Product Partnerships<br>SMS<br>Customer Lifecycle, Identity and Permissions</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Reporting<br>Intelligence<br>User Targeting<br>IAM<br>Channels<br>FIX</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Ingestion<br>Core Objects<br>Core Messaging<br>Messaging and Automation<br>Email (Composition and Infrastructure)</td>
  </tr>
</table>
</details>

### CODE REVIEW

File: _docs/_partners/message_personalization/dynamic_content/stylitics.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+---
+nav_title: Stylitics
+article_title: Stylitics
+
+description: ""
```
Comment (KellieHawks) [inline _docs/_partners/message_personalization/dynamic_content/stylitics.md:5]:
```suggestion
description: "This reference article outlines the partnership between Braze and Stylitics, a cloud-based SaaS platform that allows you to enhance your existing email campaigns with engaging and relevant bundled content, creating a personalized customer experience."
```

File: _docs/_partners/message_personalization/dynamic_content/stylitics.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+- Abandoned cart emails 
+- Abandoned browse emails 
+- Shipping confirmation emails
+- Post-purchase emails 
+
+## Integration
+
+Stylitics provides bundle data for this integration. Your email service provider can create or update the email template to include Stylitics bundles. Stylitics cannot alter the layout or design of the emails. 
+
+1. Integrate the bundle into the email. ESP determines the position and customization
```
Comment (bre-fitzgerald) [inline _docs/_partners/message_personalization/dynamic_content/stylitics.md:39]:
```suggestion
1. Integrate the bundle into the email. ESP determines the position and customization.
```

File: _docs/_partners/message_personalization/dynamic_content/stylitics.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+You or your ESP can make an API call per item to populate bundle data into your email. Stylitics recommends you use their API to make API calls as it is ready to use immediately.
+
+{% alert note %}
+If you run a Stylitics-run A/B test, the `styliticsCID` and `styliticsoverride` parameters must be appended to the PDP URLs of the Stylitics items the user clicks in the email.
+<br><br>
+For example, {% raw %}`&styliticsoverride=001?styliticsCID=email[clientname]`{% endraw %}
+{% endalert %}
+
+### 2. Flat file approach
+You of your ESP can reference an item's bundle data in a flat file to populate bundle data into your email. Stylitics can flatten bundle data into CSV, TXT, or XML format and send it to you daily. They can also help adjust the file format per your ESP's needs. Note that this takes 2-3 weeks to create this file.
```
Comment (bre-fitzgerald) [inline _docs/_partners/message_personalization/dynamic_content/stylitics.md:60]: Is this supposed to be or? 🤔 
```suggestion
You or your ESP can reference an item's bundle data in a flat file to populate bundle data into your email. Stylitics can flatten bundle data into CSV, TXT, or XML format and send it to you daily. They can also help adjust the file format per your ESP's needs. Note that this takes 2-3 weeks to create this file.
```

File: _docs/_partners/message_personalization/dynamic_content/stylitics.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+## Email template best practices 
+
+You and your ESP will create an HTML email template to insert Stylitics data and bundles. Here are some best practices and recommendations. 
+- Display 2-4 bundles in the email for the most expensive item or first full-price item the user bought or interacted with 
+- Call multiple `item_numbers` and show the first few bundle response 
+- Have a fallback option if there are no bundles available for the item 
+	- Hide the section where Stylitics bundles live 
+	- Show bundles for the next item the user viewed 
+- Displaying bundle images and a list of product titles and thumbnail images to ensure the user has a clear click-through.
```
Comment (bre-fitzgerald) [inline _docs/_partners/message_personalization/dynamic_content/stylitics.md:78]: Suggest removing gerund to match format of other bullets
```suggestion
- Display bundle images and a list of product titles and thumbnail images to ensure the user has a clear click-through
```

**Review by bre-fitzgerald — CHANGES_REQUESTED** [review]
Looks good! Few minor callouts

**Pull request merged**